images/virt-artifact/werf.inc.yaml

```diff
@@ -89,46 +89,46 @@ shell:
     - gcc -static cmd/container-disk-v2alpha/main.c -o /kubevirt-binaries/container-disk
 
     - echo ============== Build virt-launcher ====================
-    - CGO_ENABLED=1 go build -o /kubevirt-binaries/virt-launcher ./cmd/virt-launcher/
+    - CGO_ENABLED=1 go build -ldflags="-s -w" -o /kubevirt-binaries/virt-launcher ./cmd/virt-launcher/
 
     - echo ============== Build virt-handler =====================
-    - CGO_ENABLED=1 go build -o /kubevirt-binaries/virt-handler ./cmd/virt-handler/
+    - CGO_ENABLED=1 go build -ldflags="-s -w" -o /kubevirt-binaries/virt-handler ./cmd/virt-handler/
     
     - echo ============== Build virt-launcher-monitor ============
-    - go build -o /kubevirt-binaries/virt-launcher-monitor ./cmd/virt-launcher-monitor/
+    - go build -ldflags="-s -w" -o /kubevirt-binaries/virt-launcher-monitor-orig ./cmd/virt-launcher-monitor/
```

Comment:
```suggestion
    # virt-launcher-monitor is wrapped in the final image. Add suffix here to prevent image size increasing as effect of file renaming.
    - go build -ldflags="-s -w" -o /kubevirt-binaries/virt-launcher-monitor-orig ./cmd/virt-launcher-monitor/
```